TASVideos.Data/Entity/User.cs

```diff
@@ -207,6 +207,9 @@ public static IQueryable<User> ForUser(this IQueryable<User> query, string? user
 	public static IQueryable<User> ThatHaveCustomLocale(this IQueryable<User> query)
 		=> query.Where(u => u.DateFormat != UserDateFormat.Auto || u.TimeFormat != UserTimeFormat.Auto || u.DecimalFormat != UserDecimalFormat.Auto);
 
+	public static IQueryable<User> ThatAreBanned(this IQueryable<User> query)
+		=> query.Where(u => u.BannedUntil.HasValue && u.BannedUntil > DateTime.UtcNow); // > and < in these methods, but what about ==?
```

Comment:
I don't understand the comment here, if you are asking how something works, would be best not to do it in code but in a discussion

Reply:
Just an edge case I noticed in the code below this. Since it's timestamp-based, if it did ever happen, you could just reload the page.